tests/integration_tests/test_experiment.py

```diff
@@ -77,6 +80,34 @@ def test_experiment_seq_seq1(csv_filename):
         run_experiment(input_features, output_features, data_csv=rel_path)
 
 
+def test_experiment_seq_seq1_model_def_file(csv_filename, yaml_filename):
+    # seq-to-seq test to use model definition file instead of dictionary
+    input_features = ('[{name: utt, type: text, reduce_output: null, '
```

Comment:
Would it make sense to create a util to convert a dictionary to a yaml string so instead of declaring strings we can declare dictionaries (more readable) and then convert them to strings using a util function.

Reply:
great point. let me try.